gui/src/components/Taipy/DateSelector.spec.tsx

```diff
@@ -143,7 +143,7 @@ describe("DateSelector Component", () => {
             await userEvent.type(input, "{ArrowLeft}{ArrowLeft}{ArrowLeft}01012001", { delay: 1 });
             expect(dispatch).toHaveBeenLastCalledWith({
                 name: "",
-                payload: { value: "2001-01-01T00:00:00.000Z" },
+                payload: { value: "Mon Jan 01 2001" },
```

Comment:
should we not format date as YYYMMDD ?

Reply:
Indeed we should be using ISO 8601 (=> "YYYYMMDD") whenever a date is serialized

Reply:
this is the dafault format of toDateString(). if you think it is better to change it to another format then I will do it :D

Reply:
indeed toDateString is always in english ...
If python knows how to parse this by default, I think that we're good

Reply:
It knows pretty well. Tested it and it works like a charm